schemas/biomarkers.json

```diff
@@ -0,0 +1,481 @@
+           "Not applicable",
+           "Unknown"
+        ],
+        "script": ""
+      },
+      "meta": {
+        "displayName": "Progesterone Receptor Allred Score"
+      }
+    },
+    {
+      "description": "Indicate the immunohistochemical test result that refers to the overexpression or lack of expression of PD-L1 (programmed death ligand 1) in a tissue sample of a primary or metastatic malignant neoplasm. (Reference NCIt: C122807)",
+      "name": "pd-l1_status",
+      "valueType": "string",
+      "restrictions": {
+        "codelist": [
+           "Cannot be determined",
+           "Negative",
+           "Not applicable",
+           "Positive",
+           "Unknown"
+        ]
+      },
+      "meta": {
+        "displayName": "PD-L1 Status"
+      }
+    },
+    {
+      "description": "Indicate the expression of anaplastic lymphoma receptor tyrosine kinase (ALK) as assessed by immunohistochemistry (IHC).",
+      "name": "alk_ihc_status",
+      "valueType": "string",
+      "restrictions": {
+        "codelist": [
+           "Cannot be determined",
+           "Negative",
+           "Not applicable",
+           "Positive",
+           "Unknown"
+        ]
+      },
+      "meta": {
+        "displayName": "ALK IHC Status"
+      }
+    },
+    {
+      "description": "Indicate the intensity of anaplastic lymphoma receptor tyrosine kinase (ALK) as assessed by immunohistochemistry (IHC).",
+      "name": "alk_ihc_intensity",
+      "valueType": "string",
+      "restrictions": {
+        "codelist": [
+           "0 (No stain)",
+           "+1",
+           "+2",
+           "+3"
+        ],
+        "script": ""
+      },
+      "meta": {
+        "displayName": "ALK IHC Intensity"
+      }
+    },
+    {
+      "description": "Indicate the expression of anaplastic lymphoma receptor tyrosine kinase (ALK) as assessed by flurescence in situ hybridization (FISH).",
+      "name": "alk_fish_status",
+      "valueType": "string",
+      "restrictions": {
+        "codelist": [
+           "Cannot be determined",
+           "Negative",
+           "Not applicable",
+           "Positive",
+           "Unknown"
+        ]
+      },
+      "meta": {
+        "displayName": "ALK FISH Status"
+      }
+    },
+    {
+      "description": "Indicate the expression of receptor lymphoma kinase (ROS1) as assessed by immunohistochemistry (IHC).",
+      "name": "ros1_ihc_status",
+      "valueType": "string",
+      "restrictions": {
+        "codelist": [
+           "Cannot be determined",
+           "Negative",
+           "Not applicable",
+           "Positive",
+           "Unknown"
+        ]
+      },
+      "meta": {
+        "displayName": "ROS1 IHC Status"
+      }
+    },
+    {
+      "description": "Indicate the expression of Pan-TRK as assessed by immunohistochemistry (IHC).",
+      "name": "pan-trk_ihc_status",
+      "valueType": "string",
+      "restrictions": {
+        "codelist": [
```

Comment:
genral comment - should this be one common codelist?

Reply:
Good point Rosi. I aggregated this into a common codelist (`expression_status`)